cql/DashboardLibrary.cql

```diff
@@ -162,6 +167,20 @@ define function CytologyInterpretation(conclusions List<System.Concept>):
         when c ~ "ASC-H" then {text: 'ASC-H', rank: 5}
         when c ~ "AGC" then {text: 'AGC', rank: 6}
         when c in "HSIL" or c ~ "SCC" then {text: 'HSIL+', rank: 7}
+        when c ~ "AGC Favor Neoplasia" then {text: 'AGC-FN', rank: 8}
+        when c ~ "Endocervical Cells Favor Neoplasia" then {text: 'AEC-FN', rank: 8}        
+        when c in "AIS" then {text: 'AIS', rank: 9}
+
+        // Rare cytologies added so they are not interpreted as UNK, but ranked low, to-do: confirm names
+        when c ~ "Endocervical AIS" then {text: 'Endocervical AIS', rank: 1}
+        when c ~ "Histiocytes" then {text: 'Histiocytes', rank: 1}
+        when c ~ "Atypical Endocervical Cells" then {text: 'Atypical Endocervical Cells', rank: 1}
+        when c ~ "Atypical Endometrial Cells" then {text: 'Atypical Endometrial Cells', rank: 1}
+        when c ~ "Benign Endometrial Cells" then {text: 'Benign Endometrial Cells', rank:1}
+        when c ~ "Unsatisfactory" then {text: "Unsatisfactory", rank: 1}
+        when c ~ "Absent Transformation Zone" then {text: 'Absent Transformation Zone', rank: 1}
+        when c ~ "Atypical Glandular Cells Uncertain" then {text: 'AGC-US', rank:1}
```

Comment:
Where is `Atypical Glandular Cells Uncertain` used in the CQL?

Reply:
Open question - @mrnosal. I also am not sure of how well `AGC-US` will be known by the average user of our app.

Reply:
I believe AGC-US is accepted abbreviation for atypical glandular cells of undetermined significance. Will update if needed after feedback from pilots.